### DISCUSSION TITLE
Costume Folders

### DISCUSSION THREAD

**Discussion by Yellow-Brain** [ScratchAddons/ScratchAddons#5113] [Ideas] [answered]
They'd work the sam as the already implemented sprite folders, but in my opinion be'd more useful.  Uses for this could for animators (new folder for every scene) or just cleaning up projects with big counts of costumes.

**Comment by Platformer444** [comment]
We Do have Costume folders

**Reply by Yellow-Brain** [reply]
Oh

**Comment by BroJac5246** [comment, marked as answer]
The Sprite Folders addon can do this, although it's not super obvious unless you read the description :)

**Reply by Yellow-Brain** [reply]
Oh! Thank you :)
